### PR TITLE
Setup desktop

### PR DESCRIPTION
**Pull request by allenkays**
In this milestone I:
Added media queries to style for desktop.
Removed background images and linked them using css.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,25 +5,35 @@
...
             </nav>
+            <nav class="menu-desktop">
+                <a class="logo" href="#">Allan K. </a>
+                <ul class="nav-links">
+                    <li class="nav-link"><a href="#portfolio">Portfolio</a></li>
+                    <li class="nav-link"><a href="#about">About</a></li>
+                    <li class="nav-link"><a href="#feedback">Contact</a></li>
+                    <li><a href="#"><img src="assets/header/Icon - Mail.svg" alt="mail"></a></li>
+                </ul>
+            </nav>
```
Comment (usorfaitheloho) [inline index.html:26]: kindly ensure the logo is in the same vertical alignment with the headline section as it is in the design given so you'd have a good design 😄 . Also, I'd suggest you move the navigation items away from the border line a bit by adding some margin 👍 

Figma design :
image

Your design :
image

File: index.html [hunk 2]
```diff
@@ -5,25 +5,35 @@
...
+                    <li><a href="#"><img src="assets/header/Icon - Mail.svg" alt="mail"></a></li>
+                </ul>
+            </nav>           
         </header>
         <main class="content">
             <section class="welcome">
-                <article>
-                    <h1 class="main-title">Hey There. I'm Allan.</h1>
+                <article class="hello">
+                    <h1 class="main-title"><span>Hey There. I'm Allan.</h1>span></h1>
```
Comment (usorfaitheloho) [inline index.html]: Kindly add a closing tag to the `span` element on line 31 👍

File: index.html [hunk 3]
```diff
@@ -46,181 +56,249 @@ <h2 class="sub-title">I'm a Software Developer</h2>
...
+                            <p class="paragraph">
+                                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+                            </p>
+                            <ul class="stack">
+                                <li><button class="css-c" type="button">CSS</button></li>
+                                <li><button class="html-c" type="button">HTML</button></li>
+                                <li><button class="bootstrap-c" type="button">Bootstrap</button></li>
+                                <li><button class="ruby-c" type="button">Ruby</button></li>
+                            </ul>
+                            <button class="s-button" type="button">See project</button>
```
Comment (usorfaitheloho) [inline index.html:78]: Kindly add some margin-top to this section of your project, so it doesn't look clustered 😄 

Your design :
image

File: index.html [hunk 4]
```diff
@@ -46,181 +56,249 @@ <h2 class="sub-title">I'm a Software Developer</h2>
...
                             <span class="stack-title">
                                 <img src="assets/about/Rectangle_57.png" alt="shape">
                                 <h3>Skills</h3>
+                                <div class="lang-frameworks3">
+                                    <li class="stack3">Database Management</li>
+                                    <li class="stack3">Version Control</li>
+                                    <li class="stack3">CLI</li>
+                                    <li class="stack3">API Design</li>
+                                    <li class="stack3">Web Development</li>
+                                </div>
```
Comment (usorfaitheloho) [inline index.html]: Kindly note that their seems to be an issue with this section as the technologies section is missing which is close to the contact-me section. Please check your design and fix it.

Your design :
image

Figma design :

image

File: index.html [hunk 5]
```diff
@@ -236,6 +314,8 @@ <h3>Skills</h3>
                         <button class="c-button" type="submit">Get in touch</button>
                     </form>
```
Comment (usorfaitheloho) [inline index.html:234]: Also, for your contact section, kindly fix it so it is as close to the design as possible 👍 

Your design :
image

Figma design :

image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @allenkays  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * All linter checks passed ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 6]
```diff
@@ -44,17 +34,24 @@ body {
...
+.mobile-menu,
+.menu-desktop {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 30px;
+  color: #fff;
+  margin-left: 12%;
+  margin-right: 12%;
 }
```
Comment (KanzaTahreem) [inline style.css]: - [ ] Good job so far 👍 However, the logo is still not aligned with the headline section 😓 Kindly make sure to align the logo with the headline section as suggested [here](https://github.com/allenkays/Portfolio-setup-and-mobile-first/pull/4#discussion_r1264563670) for getting a design close to the figma template as much as possible ✨📌

image

File: index.html [hunk 7]
```diff
@@ -5,25 +5,35 @@
...
+                    <li><a href="#"><img src="assets/header/Icon - Mail.svg" alt="mail"></a></li>
+                </ul>
+            </nav>           
         </header>
         <main class="content">
             <section class="welcome">
-                <article>
-                    <h1 class="main-title">Hey There. I'm Allan.</h1>
+                <article class="hello">
+                    <h1 class="main-title"><span>Hey There. I'm Allan.</span></h1>span></h1>
```
Comment (KanzaTahreem) [inline index.html]: - [ ] As [suggested](https://github.com/allenkays/Portfolio-setup-and-mobile-first/pull/4#discussion_r1264563862), kindly make sure to fix the `span` tag. Right now, I can see that `span` tag is rendering as 👇
 
image

File: style.css [hunk 8]
```diff
@@ -603,15 +551,18 @@ button:disabled {
...
-  margin-top: 150px;
-  margin-bottom: 120px;
+  justify-content: center;
   align-items: center;
   color: #fff;
+  padding-top: 120px;
+  padding-bottom: 10%;
+  padding-left: 0;
   gap: 24px;
 }
```
Comment (KanzaTahreem) [inline style.css]: - [ ] Good job for adding the technologies section as [suggested](https://github.com/allenkays/Portfolio-setup-and-mobile-first/pull/4#discussion_r1264574899) 🥇🥇 However, it is not looking close to the Figma template. Kindly make sure that after adding the section, it looks close to the Figma template as much as possible  ⛏🤓

image

File: style.css [hunk 9]
```diff
@@ -636,10 +590,10 @@ button:disabled {
...
 .feedback {
   display: inline-flex;
-  padding: 120px 24px;
+  padding-top: auto;
   flex-direction: column;
-  align-items: flex-start;
-  gap: 58px;
+  justify-content: center;
+  align-items: center;
 }
```
Comment (KanzaTahreem) [inline style.css:583]: - [ ] Good job with the contact section so far 👍👍 Kindly make sure that it looks almost identical to the given Figma template as much as possible as [suggested](https://github.com/allenkays/Portfolio-setup-and-mobile-first/pull/4#discussion_r1264575018). Please fix this as well 😃
- [ ] Also, I noticed that **_First name and Last name_** labels are present in the contact form. Previously only the label for **_Full name_** was present for the name field. Kindly fix this also ⛏💪 In the last review, the reviewer didn't asked for adding the two input fields for name as it is given in the project requirements to use only one input field for it. However, kindly fix the design in terms of padding/margin/space and alignment for getting a design close to the Figma template as requested earlier 💪

image

**Review by KanzaTahreem — CHANGES_REQUESTED** [review]
# Changes Required 🔁

Hi @allenkays 

Great work on making the changes requested by a previous reviewer 👏! I really appreciate the effort you put in!  **_However, there are few this that still need to be fixed. But you are almost there!_** 💪 🏁

so close


## Highlights 🎯

There are some excellent points that you have made perfectly. Keep up the great work! 👏
- ✅ Good job with the design
- ✅ Great job with CSS
- ✅ Descriptive PR and README file
- ✅ Correct Linters

## Required Changes 🛑

_Please check the comments under the review._

## Optional suggestions ❕

N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AbelG101 — COMMENTED** [review]
# Hi @allenkays 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- Please take note that since you're repeating the week with an unapproved PR, you'll have to submit a new PR for review as per the policies in Microverse

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Pull request closed without merging**